CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### New features
 
+* [#8810](https://github.com/rubocop-hq/rubocop/pull/8810): Fix multiple offense detection for Style/RaiseArgs. ([@pbernays][])
```

Comment:
Can you create `### Bug fixes` header and move this entry there?

```diff
 ### New features

 * [#8796](https://github.com/rubocop-hq/rubocop/pull/8796): Add new `Lint/HashCompareByIdentity` cop. ([@fatkodima][])
+
+### Bug fixes
+
+* [#8810](https://github.com/rubocop-hq/rubocop/pull/8810): Fix multiple offense detection for Style/RaiseArgs. ([@pbernays][])

 ## 0.92.0 (2020-09-25)
```

Reply:
Can do

Reply:
Thank you for the updating.
One more point. Can you rebase with the latest master branch to resolve the changelog conflict?

Reply:
👍